DenunciaCiudadana/src/assets/styles/stylesForm.js

```diff
@@ -6,6 +6,7 @@ const stylesForm = StyleSheet.create({
     color: '#007aff',
     fontSize: 28,
     fontWeight: 'bold',
+    textAlign: 'center',
   },
   text: {
     color: '#121212',
```

Comment:
Someone said you to use constants in colors and just call it... I recommend to do it